e2e/integration/ssi.js

```diff
@@ -0,0 +1,55 @@
+import { apps } from "../helper/apps.list";
+context( 'Test for SSI', () => {
+
+    before( () => {
+        cy.visit( Cypress.env( 'STAGE_HOST' ) );
+        Cypress.config( 'includeShadowDom', true );
+        cy.get( '#username', { timeout: 5000 } ).type( Cypress.env( 'USERNAME' ) );
+        cy.get( '#password' ).type( Cypress.env( 'PASSWORD' ) );
+        cy.get( '#submit' ).click();
+    } );
+    Cypress.on( 'uncaught:exception', ( err, runnable ) => {
+        return false;
+    } );
```

Comment:
`return false` from this handler will not let Cypress fail the tests. So do our tests pass all the time. @supriyagawade Can you check if this should be necessary here?

Reply:
@ghanlohar That seems to be a common fix for any uncaught exceptions that might occur in the app. It's not for the test cases.

https://stackoverflow.com/questions/53845493/cypress-uncaught-assertion-error-despite-cy-onuncaughtexception